mirrors_countme/scripts/countme_totals.py

```diff
@@ -22,7 +22,7 @@ def parse_args(argv=None):
         "--update-from",
         metavar="COUNTME_RAW_DB",
         dest="countme_raw",
-        help="Update totals from raw data (from ./parse-access-log.py)",
+        help="Update totals from raw data (from ./countme_parse_access_log.py)",
```

Comment:
Good catch! Please make it plain `coutme_parse_access_log`, though –  the scripts will be in the PATH without the `.py` suffix with #57/#60.

Reply:
done